packages/xo-web/src/xo-app/host/tab-advanced.js

```diff
@@ -116,6 +118,20 @@ export default class extends Component {
     () => this.props.host.CPUs.flags,
     flags => /\bht\b/.test(flags)
   )
+  _setHostIscsiIqn = iscsiIqn =>
+    confirm({
+      icon: 'alarm',
+      title: _('editHostIscsiIqnTitle'),
+      body: (
+        <span>
+          {_('editHostIscsiIqnMessage')}
+          <br />
+          {_('editHostIscsiIqnInfo')}
+        </span>
+      ),
+    })
+      .then(() => editHost(this.props.host, { iscsiIqn }))
+      .catch(noop)
```

Comment:
`.catch` is not needed.

Reply:
We need it because the [onBlur](https://github.com/vatesfr/xen-orchestra/blob/master/packages/xo-web/src/common/editable/index.js#L279)  still triggers  [onChange](https://github.com/vatesfr/xen-orchestra/blob/master/packages/xo-web/src/common/editable/index.js#L107) if we cancel. (`editable/Text`)

Reply:
`then(() => editHost(this.props.host, { iscsiIqn }), noop)`